src/digraph.rs

```diff
@@ -1720,6 +1720,25 @@ impl PyDiGraph {
         }
     }
 
+    /// Get the direction-agnostic neighbors (i.e. successors and predecessors) of a node.
+    ///
```

Comment:
Can you add an example use on the docstring? I was thinking something along the lines of calling this method and comparing to calling `to_undirected` in the whole graph and then calling `neighbors`

Reply:
Sounds good! Added in 2824fe4 🙂